2.8.0

```diff
@@ -0,0 +1,9 @@
++ @babel/preset-env@7.1.6
```

Comment:
what is the purpose of this file. Why we add this

Reply:
there are some problems while running react-native (before your last config), so during that time I add that file. I will delete it later.